services/ui-src/src/components/reports/ReportPageIntro.tsx

```diff
@@ -14,57 +14,43 @@ export const ReportPageIntro = ({
   reportYear,
   ...props
 }: Props) => {
-  const { section, subsection, hint, info } = text;
+  const { section, subsection, info, introAccordion } = text;
 
-  if (section && (subsection || initiativeName)) {
+  const isSubsectionPage = section && (subsection || initiativeName);
+
+  const header = () => {
     return (
-      <Box sx={sx.introBox} {...props}>
+      <Heading as="h1" sx={sx.largeHeading}>
+        {subsection}
+      </Heading>
+    );
+  };
+
+  const headerSubsection = () => {
+    return (
+      <>
         <Heading as="h1" sx={sx.smallHeading}>
           {section}
         </Heading>
         <Heading as="h2" sx={sx.largeHeading}>
           {initiativeName ? initiativeName : subsection}
```

Comment:
I know it's out of scope, but why couldn't this be `initiativeName || subsection` or `initiativeName ?? subsection`?

Reply:
I like it. Gonna make you all take another look on Monday after I push up a fix 😆